plugins/inputs/kafka_consumer/sample.conf

```diff
@@ -6,6 +6,14 @@
   ## Topics to consume.
   topics = ["telegraf"]
 
+  ## Topic regular expressions to consume.  Matches will be added to topics.
+  # topic_regexps = [ "*test", "metric[0-9A-z]*" ]
```

Comment:
I'm not sure this is the best name. Maybe something like `dynamic_topic_filter` is more speaking?

Reply:
We might want to make `topic_refresh_interval` `dynamic_topic_refresh_interval`, but I think it's a plausible use case to write a regular expression that matches many available topics, just to simplify your configuration, and not refresh it dynamically.   It really *is* a list of regular expressions to match to available topics.

Maybe I should not abbreviate it, but I think anyone who wants to use a regular expression is going to know what `regexp` means.